Marlin/src/lcd/extui/lib/mks_ui/SPI_TFT.cpp

```diff
@@ -22,15 +22,15 @@
 
 #include "../../../../inc/MarlinConfigPre.h"
 
-#if ENABLED(TFT_LVGL_UI_SPI)
```

Comment:
Even though the new PlatformIO scripts remove the need to wrap CPP files, we should still retain these wrappers for the benefit of other potential build systems.